### ISSUE TITLE
Reboot rig schedule

### ISSUE THREAD

**Issue by astronyu** [RainbowMiner/RainbowMiner#1222]
@RainbowMiner , something weird happened today. It seems my rig has been frozen for 2 months! 
Local dashboard shows the rig is mining on NH but when I checked NH, there wasn't any mining activity and the last payment was 2 months ago!

So, I restarted RBM and then it started showing notice that my version is now updating from 4.6.2.1 and finally NH and MPH showing there is mining activities.

Is there a way to have the rig rebooted at schedule? or check if the mining froze and reboot the script/rig if it does?

[debug_2020-11-01.zip](https://github.com/RainbowMiner/RainbowMiner/files/5470548/debug_2020-11-01.zip)

**Comment by RainbowMiner** [comment]
Now that sounds weird!
Did you by any chance click into the console window back then? That can hold the script operation.
I recommend using the monitoring central on https://rbminer.net - you would have got a push message (or email) with a rig-is-down message.

I will add an extra watchdog process, that will reboot the Maschine, in case the main script halts for a long time.

**Comment by astronyu** [comment]
Yeah, it is weird. When I check monitoring central, there wasn't any stats and I didn't received any push message (normally I do).

**Comment by astronyu** [comment]
@RainbowMiner I think it happened again but not sure since in NH the status showing the miner is mining.

FYI, I had to reboot the PC and reinstall all my GPU after updating motherboard's BIOS. Then, I started mining and it seems like everything was going fine. Only that now it mining consistently to NH and didn't switched to MPH like usual.

Check rbminer.net and there wasn't any stats shown. The local dashboard shows it has been mining consistently for 10 hours although only on NH and didn't switched to MPH like it usually do.

[debug_2020-11-26.zip](https://github.com/RainbowMiner/RainbowMiner/files/5599037/debug_2020-11-26.zip)
